README.md

```diff
@@ -206,29 +115,19 @@ Example:
 
 > Mention all of the collaborators of this project.
```

Comment:
- [ ] Kindly remove this line as it was just an instruction text to help you set up your README file 😉